index.html

```diff
@@ -187,6 +187,41 @@ <h2>A Special time</h2>
             <a href="" class="social-net"><i class="fas fa-share"></i></a>
           </div>
         </div>
+        <div class="comments">
+          <form>
+            <fieldset>
+              <h2>What is your overall opinion about this article?</h2>
+              <div class="flex-container overal">
+                <label for="positive"><input type="radio" id="positive" value="positive">Positive </label>
+              <label for="negative"><input type="radio" id="negative" value="negative">Negative </label>
+              <label for="neutral"><input type="radio" id="neutral" value="neutral">Neutral </label>
+              </div>
+            </fieldset>
+            <fieldset>
+              <label class="xxl-font"><input type="checkbox"> I want to subscribe to the newsletter </label>
+            </fieldset>
+            <fieldset>
+              <div class="flex-container login">
+                <h2>Log In to add a comment</h2>
+                <a href="#">Not registered</a>
+              </div>
+              <div class="flex-container login">
+                <div class="wrapper">
+                  <input class="text" maxlength="20" type="text" placeholder="Email or Username" required><br>
+                  <input class="text" maxlength="20" type="text" placeholder="Password" required><br>
```

Comment:
Please add input of type "email" labeled with text "Email" and input of type "text" labeled with text "Name", as exercise requires. 
You can keep the Password input but then you should modify the type of it to be "password" and label it with the text "Password".